app/org/sagebionetworks/bridge/models/accounts/User.java

```diff
@@ -176,11 +153,35 @@ public boolean isInRole(Roles role) {
     public boolean isInRole(Set<Roles> roleSet) {
         return roleSet != null && !Collections.disjoint(roles, roleSet);
     }
+    
+    public List<ConsentStatus> getConsentStatuses() {
+        return consentStatuses;
+    }
+    
+    public void setConsentStatuses(List<ConsentStatus> consentStatuses) {
+        this.consentStatuses = (consentStatuses == null) ? ImmutableList.of() : ImmutableList.copyOf(consentStatuses);
+    }
```

Comment:
Should we make ConsentStatuses a map from subpopGuid to consentStatus? This will make it clear that there's only one consentStatus per subpop.